components/expenses/PayoutBankInformationForm.js

```diff
@@ -219,27 +219,28 @@ const availableCurrenciesQuery = gqlV2`
 /**
  * Form for payout bank information. Must be used with Formik.
  */
-const PayoutBankInformationForm = ({ isNew, getFieldName, host }) => {
+const PayoutBankInformationForm = ({ isNew, getFieldName, host, fixedCurrency }) => {
   const { data } = useQuery(availableCurrenciesQuery, {
     context: API_V2_CONTEXT,
     variables: { slug: host.slug },
-    skip: Boolean(host.transferwise?.availableCurrencies),
+    skip: Boolean(fixedCurrency || host.transferwise?.availableCurrencies),
   });
   const formik = useFormikContext();
   const { formatMessage } = useIntl();
-  const availableCurrencies = host.transferwise?.availableCurrencies || data?.host?.transferwise?.availableCurrencies;
+  const availableCurrencies =
+    fixedCurrency || host.transferwise?.availableCurrencies || data?.host?.transferwise?.availableCurrencies;
```

Comment:
It's weird to put `fixedCurrency` in what was until now expected to be an array. We should generally avoid long chains of `||` or `&&` for variable assignment.

Reply:
This logic can also be simplified.